src/coredump/coredumpctl.c

```diff
@@ -977,26 +997,42 @@ static int run_debug(int argc, char **argv, void *userdata) {
                 return log_error_errno(SYNTHETIC_ERRNO(ENOENT),
                                        "Binary is not an absolute path.");
 
+        r = strv_extend(&debugger_call, exe);
+        if (r < 0)
+                return log_oom();
+
+        r = strv_extend(&debugger_call, "-c");
+        if (r < 0)
+                return log_oom();
+
         r = save_core(j, NULL, &path, &unlink_path);
         if (r < 0)
                 return r;
 
+        r = strv_extend(&debugger_call, path);
+        if (r < 0)
+                return log_oom();
+
+        r = strv_extend(&debugger_call, NULL);
```

Comment:
that's a NOP. strv arrays are always NULL terminated. this line has exactly no effect, since you want to append a NULL to something already NULL terminted